compiler/p/codegen/PPCBinaryEncoding.cpp

```diff
@@ -336,7 +326,7 @@ uint8_t *TR::PPCConditionalBranchInstruction::generateBinaryEncoding()
 
 int32_t TR::PPCConditionalBranchInstruction::estimateBinaryLength(int32_t currentEstimate)
    {
-   setEstimatedBinaryLength(PPC_INSTRUCTION_LENGTH);
+   setEstimatedBinaryLength(PPC_INSTRUCTION_LENGTH * 2);
```

Comment:
A comment explaining the need for doubling the length would be helpful to future readers.

Reply:
Fixed in 0fbf852bc2c3f1b74f8d67f95138e3d15ea09dfb